coordinatorlib/src/it/scala/com/socrata/datacoordinator/truth/metadata/sql/PostgresDatasetMapWriterTest.scala

```diff
@@ -29,12 +55,14 @@ class PostgresDatasetMapWriterTest extends FunSuite with MustMatchers with Befor
   }
   val ZeroID = 0L
 
+  val resourcName = Some("1234-abcc")
+
   def populateDatabase(conn: Connection) {
     Migration.migrateDb(conn)
   }
 
   def withDb[T]()(f: (Connection) => T): T = {
-    using(DriverManager.getConnection("jdbc:postgresql://localhost:5432/blist_test", "blist", "blist")) { conn =>
+    using(DriverManager.getConnection("jdbc:postgresql://localhost:5432/datacoordinator_test", "blist", "blist")) { conn =>
```

Comment:
Make it drop and create a new database every run?

Reply:
Since postgres doesn't support `create if not exists` for databases it would be a little bit more complicated logic than I really want to write for this... we expect our test postgres databases to exist for all our other tests I can think of.